src/wwwroot/template-editor/adapterService.js

```diff
@@ -340,40 +353,10 @@
     function getSettings(idCampaign, idTemplate) {
       traceAdapterCall(getSettings, arguments);
       var msEditorSettings = {
-        language: 'es',
-        sharedSocialNetworks: [{
-          name: 'twitter',
-          url: 'http://twitter.com/share?related=fromdoppler',
-          idSocialNetwork: '0'
-        }, {
-          name: 'linkedin',
-          url: 'http://linkedin.com/',
-          idSocialNetwork: '1'
-        }, {
-          name: 'facebook',
-          url: 'http://facebook.com',
-          idSocialNetwork: '2'
-        }, {
-          name: 'googlemas',
-          url: 'http://google.com',
-          idSocialNetwork: '3'
-        }, {
-          name: 'pinterest',
-          url: 'http://pinterest.com',
-          idSocialNetwork: '4'
-        }, {
-          name: 'whatsapp',
-          url: 'http://whatsapp.com',
-          idSocialNetwork: '5'
-        }],
-        stores: [{
-          name: 'TiendaNube',
-          accessToken: '4b9883131f01fbde3f168dde5bca209b1db1fbb1',
-          storeId: '404554'
-        }],
-        redirectUrl: '/MSEditor/Editor?idCampaign=1&redirectedFromSummary=True',
-        rssCampaign: true,
-        rssShowPreview: true
+        language: langKey,
+        redirectUrl: '#/templates',
```

Comment:
I think that it will not work because you have to change the URL not only the hash